client/extensions/woocommerce/app/settings/email/email-settings/components/internal-notification.js

```diff
@@ -61,14 +61,14 @@ const InternalNotification = ( {
 			<ListItemField className="components__notification-component-input">
 				<FormTextInput
 					className={ isPlaceholder ? 'components__is-placeholder' : null }
-					isError={ emailValidationError }
+					isError={ checked && emailValidationError }
 					name={ item.field }
 					onChange={ change }
 					value={ recipient }
-					placeholder={ placeholder }
+					placeholder={ isPlaceholder ? '' : translate( 'Recipient email address.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Recipient's username or email address' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).